WebKitBrowser/WebKitBrowser.cpp

```diff
@@ -202,32 +209,41 @@ namespace Plugin {
 
     void WebKitBrowser::LoadFinished(const string& URL, int32_t code)
     {
-        TRACE(Trace::Information, (_T("LoadFinished: { \"url\": \"%s\", \"httpstatus\": %d}"), URL.c_str(), code));
+        string message(string("{ \"url\": \"") + URL + string("\", \"loaded\":true }"));
+        TRACE(Trace::Information, (_T("LoadFinished: %s"), message.c_str()));
+        _service->Notify(message);
         event_loadfinished(URL, code);
         URLChange(URL, true);
     }
 
     void WebKitBrowser::LoadFailed(const string& URL)
     {
-        TRACE(Trace::Information, (_T("LoadFailed: { \"url\": \"%s\" }"), URL.c_str()));
+        string message(string("{ \"url\": \"") + URL + string("\" }"));
+        TRACE(Trace::Information, (_T("LoadFailed: %s"), message.c_str()));
+        _service->Notify(message);
         event_loadfailed(URL);
     }
 
     void WebKitBrowser::URLChange(const string& URL, bool loaded)
     {
-        TRACE(Trace::Information, (_T("URLChange: { \"url\": \"%s\", \"loaded\": \"%s\" }"), URL.c_str(), (loaded ? "true" : "false")));
+        string message(string("{ \"url\": \"") + URL + string("\" }"));
```

Comment:
Agree.

Reply:
We cannot change the "URLChanged" into "URLChange" for the notify, that would break backwards compatibility. We'll add the loaded here as well,  adding info is not a problem.